CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2021-10-08
+
+This release is rolling out to all regions - estimated time for completed roll out is 2021-10-14 for public cloud and 2021-10-21 for sovereign clouds.
+
+### Release Notes
+
+* Features
+  * General Availability of Ultra SSD support
```

Comment:
worth adding a link to the docs. https://docs.microsoft.com/en-us/azure/aks/use-ultra-disks